src/service-worker.11ty.js

```diff
@@ -0,0 +1,42 @@
+const glob = require("glob")
+
+/**
+ * Generates the service worker that browsers use for app-level caching.
+ * Only caches images because they are responsible for the majority of the bandwidth usage but rarely change.
+ * When an existing image changes, update the `version` below, or else past visitors will only see the old image.
```

Comment:
I want this to automatically generate the list of files to cache so that new content is easy to add without being missed in the cache. Even having a linter enforce this would be too much of a pain IMO, especially for those who don't do web development regularly.